tests/infra/ccf.py

```diff
@@ -421,14 +421,59 @@ def open_network(self, node):
         proposal and make members vote to transition the network to state
         OPEN.
         """
-        result = self.propose(1, node, "open_network")
-        self.vote_using_majority(node, result[1]["id"])
-        self.check_for_service(node)
+        if os.getenv("HTTP"):
```

Comment:
I think we should modify the lower-level functions here (i.e. `propose()` and `vote()`) instead of this higher-level function.

Reply:
Yes, agreed, the whole suite should be replaced (and moved out). I'm going to do that once I've got the signatures fixed.